subprojects/docs/src/docs/userguide/jvm/jacoco_report_aggregation_plugin.adoc

```diff
@@ -0,0 +1,23 @@
+// Copyright 2021 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[jacoco_report_aggregation_plugin]]
+= The JaCoCo Report Aggregation Plugin
+
+The JaCoCo Report Aggregation plugin (plugin id: `jacoco-report-aggregation`) provides tasks and configurations which can be used to aggregate the results of multiple JaCoCo code coverage reports into a single HTML report.  These coverage reports are a side effect of link:{groovyDslPath}/org.gradle.api.tasks.testing.Test.html[Test] task invocations; see more at the <<jacoco_plugin#jacoco_plugin,JaCoCo Plugin>> chapter. This aggregation is done in a way that is compatible with the <<configuration_cache#configuration_cache,configuration cache>>.
```

Comment:
I don't like the phrase "side effect" here, and think we should be precise about the different between the binary coverage data and the human-readable jacoco report - accidentally conflating things like this can really cost newcomers some time.  The reports aren't the result of running Test, the coverage results binary is.  

Maybe change "code coverage reports" in the first sentence to "inspections"? and do something for the second sentence like: "Coverage data will be automatically generated by Test task invocations in a project where the jacoco plugin has been applied."

Reply:
Good ideas; I've applied your suggestions